routes/v1.js

```diff
@@ -97,6 +101,14 @@ function addRoutes(app, peliasConfig) {
   const placeholderService = serviceWrapper(placeholderConfiguration);
   const isPlaceholderServiceEnabled = _.constant(placeholderConfiguration.isEnabled());
 
+  const changeLanguageConfiguration = new Language(_.defaultTo(peliasConfig.api.services.language, {}));
```

Comment:
Just that they are pointing to the same service. But I guess the URLs and results are different enough to warrant separate services. We also talked about combining the Pelias config parameters into one service instead of having both language and placeholder.